k256/src/arithmetic/scalar.rs

```diff
@@ -172,7 +181,7 @@ impl PrimeFieldBits for Scalar {
     }
 
     fn char_le_bits() -> ScalarBits {
-        MODULUS.into()
+        MODULUS_ARR.into()
```

Comment:
Ideally this would be `MODULUS.to_uint_array().into()` but it can't be yet because of the `force-32-bit` feature.